imagery/i.evapo.time/i.evapo.time.html

```diff
@@ -50,6 +50,21 @@ <h2>NOTES</h2>
 For multi-year calculations, just continue incrementing DOY values above
 366, it will continue working, up to maximum input of 400 satellite images.
 
+<p>
+<img src="i_evapo_time.png" alt="Temporal integration from a weather station"><br>
```

Comment:
As previously noted, the image wasn't committed in this PR. Could you please commit it and push it to your branch? After that, we will be able to continue reviewing it.

If a script or command was used to generate that image, could you write it here or somewhere so we could be able to recreate it in the future if needed? (Assuming it is a simple example)